src/main/java/net/sf/jabref/JabRef.java

```diff
@@ -350,7 +350,7 @@ public void start(String[] args) {
                     }
                     BibDatabaseContext databaseContext = pr.getDatabaseContext();
                     databaseContext.setDatabaseFile(theFile);
-                    Globals.prefs.fileDirForDatabase = databaseContext.getFileDirectory(Globals.FILE_FIELD).toArray(new String[0]);
+                    Globals.prefs.fileDirForDatabase = databaseContext.getFileDirectory().toArray(new String[0]);
```

Comment:
Yes you are right this should be a list. I even think this field should be removed completely. However, since its usage is hidden deeply in some layout logic, this will be a bigger project...